app/utils/loadDashboards.tsx

```diff
@@ -32,9 +33,10 @@ export const loadDashboards = async () => {
                             .replace('-', ' '),
                         date: stats.mtime,
                         uri: name.split('.')[0],
+                        raw: getServerHostName() + '/uploads/' + name
```

Comment:
Not sure if the server host name is needed here. The URL provided in `raw` could not be only a relative URL?

Reply:
We don´t need the server's hostname for the nextJS app itself, but we need to provide an absolute URL for users who want to integrate the React frontend components in an another website.
I'll just remove this for now until I find a better solution.